profile_bluesky/startup/instrument/devices/magnets.py

```diff
@@ -0,0 +1,38 @@
+"""
+Magnet motors
+"""
+
+__all__ = [
+    'mag6t',
+    ]
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from ophyd import Component, Device, EpicsMotor, FormattedComponent
+
+## Magnet and sample motors ##
+class Magnet6T(Device):
```

Comment:
might consider [`ophyd.MotorBundle`](https://github.com/bluesky/ophyd/blob/1edbdee8a49b376618f817aeb29db541282cf971/ophyd/epics_motor.py#L362-L367) instead of `ophyd.Device`

> provides better default behavior for `hints`.

Reply:
The _ami_controller_  branch adds a bunch of EpicsSignal components to this class, so I will leave it as a Device. But will change other places to MotorBundle.